src/fiatExchanges/SelectProvider.tsx

```diff
@@ -0,0 +1,516 @@
+      .sort(sortQuotes) || []
+  return (
+    <ScrollView>
+      <PaymentMethodSection
+        cicoQuotes={cicoQuotes}
+        paymentMethod={PaymentMethod.Card}
+        setNoPaymentMethods={setNoPaymentMethods}
+        quoteOnPress={quoteOnPress}
+      />
+      <PaymentMethodSection
+        cicoQuotes={cicoQuotes}
+        paymentMethod={PaymentMethod.Bank}
+        setNoPaymentMethods={setNoPaymentMethods}
+        quoteOnPress={quoteOnPress}
+      />
+      <LocalProviderMobileMoneySection
+        localProvider={asyncProviders.result?.availableLocalProviders[0]}
+        digitalAsset={currencyToBuy}
+      />
+      <ExchangesSection selectedCurrency={route.params.selectedCrypto} flow={flow} />
+      <LimitedPaymentMethods visible={noPaymentMethods} />
+    </ScrollView>
+  )
+}
+
+function LimitedPaymentMethods({ visible }: { visible: boolean }) {
+  const { t } = useTranslation()
+  const [isDialogVisible, setIsDialogVisible] = useState(false)
+  const dismissDialog = () => {
+    setIsDialogVisible(false)
+  }
+  const openDialog = () => {
+    setIsDialogVisible(true)
+  }
+  return (
+    <>
+      {visible && (
+        <View style={styles.noQuotesContainer}>
+          <Text style={styles.noQuotesText}>
+            {t('selectProviderScreen.somePaymentsUnavailable')}{' '}
+            <Text onPress={openDialog} style={styles.underline}>
+              {t('selectProviderScreen.learnMore')}
+            </Text>
+          </Text>
+        </View>
+      )}
+      <Dialog
+        title={t('selectProviderScreen.whyMissingPayments')}
+        isVisible={isDialogVisible}
+        actionText={t('selectProviderScreen.dismiss')}
+        actionPress={dismissDialog}
+        isActionHighlighted={false}
+        onBackgroundPress={dismissDialog}
+      >
+        <Text style={styles.dialog}>{t('selectProviderScreen.missingPaymentsExplained')}</Text>
+      </Dialog>
+    </>
+  )
+}
+
+function ExchangesSection({
+  flow,
+  selectedCurrency,
+}: {
+  flow: CICOFlow
+  selectedCurrency: Currency
+}) {
+  const { t } = useTranslation()
+  const goToExchangesScreen = () => {
+    navigate(Screens.ExternalExchanges, {
+      currency: selectedCurrency,
+      isCashIn: flow === CICOFlow.CashIn,
+    })
+  }
+  return (
+    <View style={styles.container}>
+      <Touchable onPress={goToExchangesScreen}>
+        <View style={{ ...styles.expandableContainer, paddingVertical: 27 }}>
+          <View style={styles.left}>
+            <Text style={styles.category}>{t('selectProviderScreen.cryptoExchange')}</Text>
+
+            <Text style={styles.fee}>{t('selectProviderScreen.feesVary')}</Text>
+          </View>
+
+          <View style={styles.right}>
+            <Text style={styles.linkToOtherScreen}>{t('selectProviderScreen.viewExchanges')}</Text>
+          </View>
+        </View>
+      </Touchable>
+    </View>
+  )
+}
+
+function LocalProviderMobileMoneySection({
```

Comment:
These were previously tucked in the Screen where you select your currency. I only realized it existed when digging into the code. Its only used for a few Mobile money providers in the Philippines and Brazil

Reply:
interesting find :)